docs/playbooks/browse.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 📚 Browse playbooks
+---
+
+
+<details>
+<summary>🤔 Wait, WTF are playbooks!?</summary>
+<br />
+
+Playbooks are like lessons or tutorials that help you find your way!
+
+MetaGame playbooks are quite special though as some have been written by OGs of the Metaverse! 😱
+
+If you'd like to write a playbook for MetaGame, hop into [our Discord](https://discord.gg/ZqdPP9b) and say Hi!
+
+</details>
+
+
+<p></p>
+
+
+import { PlayBooks } from "../../src/components/playbooks.js";
+
+<!-- add playbooks in this section  -->
+export const books = [
+  {
+    id: 1,
+    title: "😎 Make it Without Technical Skills",
+    author: "@cooopahtroopa",
+    imgURL: "https://i.imgur.com/BWIoQ9Q.jpg",
+    bookPath: "playbooks/how-to-make-it-without-technical-skills",
+    decription:
+      "Vestibulum ullamcorper mauris at ligula. Phasellus magna. Aenean ut eros et nisl sagittis vestibulum. Fusce fermentum. Nunc sed turpis.",
```

Comment:
Lorem ipsum. not sure if this is displayed anywhere currently

Reply:
It is not displayed anywhere currently but it's a good point. I'll remove

Reply:
fixed in 3e692ee